src/components/generic/Counter.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const StyledCounter = styled.div`
+    font-family: 'Roboto Mono', monospace;
+    font-size: 35px;
+    margin: 10px 0;
+    display: inline-block;
+    width: 200px;
+`;
+
+const Counter = ({ value }) => {
```

Comment:
you could simply export your `StyledCounter` and use it as `<Counter>{someValue}</Counter>` instead of `<Counter value={someValue}/>`